rlbot_gui/gui.py

```diff
@@ -261,7 +304,7 @@ def begin_python_bot(bot_name):
 
 def on_websocket_close(page, sockets):
     global should_quit
-    eel.sleep(3.0)  # We might have just refreshed. Give the websocket a moment to reconnect.
+    eel.sleep(100.0)  # We might have just refreshed. Give the websocket a moment to reconnect.
```

Comment:
is this 100 seconds?
isn't that like 2 minutes?

Reply:
I thought it is 100ms. At 3 the error occoured and at 5 it did. To be safe i changed it to 100